cumulusci.yml

```diff
@@ -194,6 +200,18 @@ flows:
             7:
                 task: test_data_relationships
 
+    config_acctsc_rollups:
```

Comment:
Since the Account Soft Credit feature has been merged into master, is there still a need for a flow specific to testing it? 

Though, I really like the idea of a task that creates the AccountSoftCredit fields and rollups because I think we can expand on this to create Payment and AccountContactSoftCredit rollups and target fields for use with the CRLP Validation Script.

Reply:
You may be right that it's no longer needed, but a) it'll come in handy for testing any adjustments/bug fixes to account soft credits in the future, and b) it serves as an example of how to create a flow for deploying unpackaged metadata specific to testing one feature.